src/service/standard-actions-impl.js

```diff
@@ -267,6 +269,35 @@ export class StandardActions {
       return this.handleHide(invocation);
     }
   }
+
+  /**
+   * Handles "toggleClass" action.
+   * @param {!./action-impl.ActionInvocation} invocation
+   * @return {?Promise}
+   */
+  handleToggleClass(invocation) {
+    if (!invocation.satisfiesTrust(ActionTrust.HIGH)) {
+      return null;
+    }
+
+    const target = dev().assertElement(invocation.node);
+    const {args} = invocation;
+    const className = user().assertString(args['class'],
+        "Requires argument 'class' as string for toggleClass");
+
+    this.resources_.mutateElement(target, () => {
+      if (args['force'] !== undefined) {
+        // must be boolean, won't do type conversion
+        const shouldForce = user().assertBoolean(args['force'],
+            "Requires argument 'force' as boolean for toggleClass");
```

Comment:
Nit: this error message is a little misleading--it seems to suggest that the `force` argument is required, whereas what we intend to say is that the argument is optional, but it must be a boolean. This should be different from the previous error message. Something like "Optional argument 'force' must be a boolean."